cmd/boulder-publisher/main.go

```diff
@@ -23,19 +23,17 @@ func main() {
 		go cmd.DebugServer(c.Publisher.DebugAddr)
 		go cmd.ProfileCmd("Publisher", stats)
 
-		saRPC, err := rpc.NewAmqpRPCClient("Publisher->SA", c.AMQP.SA.Server, c, stats)
-		cmd.FailOnError(err, "Unable to create SA RPC client")
-
-		sac, err := rpc.NewStorageAuthorityClient(saRPC)
+		pubConf := c.Publisher
+		amqpConf := pubConf.AMQP
```

Comment:
`amqpConf := c.Publisher.AMQP` and `c.Publisher.MaxConcurrentRPCServerRequests` below.